.env

```diff
@@ -1,6 +1,7 @@
 NODE_URL=ws://node:9944
-SMART_CONTRACT_ADDRESS=SMART_CONTRACT_ADDRESS
-APP_WALLET_JSON=APP_WALLET_JSON
-APP_WALLET_PASSPHRASE=APP_WALLET_PASSPHRASE
+SMART_CONTRACT_ADDRESS=5CgGJhgZdUHpsC6nHy3Y1iVpvtbW7EhBKvuAXE5WFsbmBVFk
```

Comment:
Please revert all changes from this file

Reply:
@nikohosn 
This is template. We should not change here anything and push

Reply:
Removed